frontend/public/components/cluster-settings/related-objects.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { sortable } from '@patternfly/react-table';
+import { Table, TableRow, TableData, RowFunctionArgs } from '../factory';
+import {
+  referenceForModel,
+  ClusterOperator,
+  ClusterOperatorObjectReference,
+  useModelFinder,
+} from '../../module/k8s';
+import { ResourceLink, EmptyBox } from '../utils';
+
+const tableColumnClasses = ['', classNames('pf-m-hidden', 'pf-m-visible-on-sm'), ''];
+
+const Header = () => [
+  {
+    title: 'Name',
+    sortField: 'name',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[0] },
+  },
+  {
+    title: 'Kind',
```

Comment:
We should probably call this "Resource" since it's not the camel-case kind string, which refers specifically to `DaemonSet` and not `daemonsets`. (I realize this is different than the design.)

cc @itsptk

Reply:
Sounds good to me